packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -43,21 +43,29 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
   public render() {
     const {
       calloutProps,
-      tooltipProps,
-      content,
       children,
+      content,
+      delay,
       directionalHint,
       directionalHintForRTL,
-      delay,
+      hostClassName,
       id,
+      overflowMode,
       setAriaDescribedBy = true,
-      hostClassName
+      tooltipProps
     } = this.props;
     const { isTooltipVisible } = this.state;
     const tooltipId = id || getId('tooltip');
+
     return (
       <div
-        className={ css('ms-TooltipHost', styles.host, hostClassName) }
+        className={ css('ms-TooltipHost',
+          styles.host,
+          hostClassName,
+          {
+            [styles.hostOverflow]: overflowMode !== undefined,
```

Comment:
instead of using a map like this, do this:

```
css(
  (condition) && styles.hostOverflow,
  ...
)
```
This will remove the various undefineds in the snapshot.